resources/ts/Components/UitIdWidget.tsx

```diff
@@ -53,7 +53,7 @@ export const UitIdWidget = ({
           },
         ],
       }),
-    [auth0Domain, profileUrl, registerUrl]
```

Comment:
Haha I noticed it as well 👍 
https://github.com/cultuurnet/publiq-platform/pull/734/commits/4b5e6a7cf8da0a80db43d122345c4bc716e21413